### PR TITLE
Header and footer updates

### PR DESCRIPTION
**Pull request by caleybrock**
# Description

Header and footer updates based on Joey's prototype:
 - settings button
 - only show home and sample test in header
 - update logos (next PR will add Canada logo)
 - footer is blue
 - start/submit button is green in bottom right aligned with notepad
 - quit test is in the header

## Type of change
- New feature (non-breaking change which adds functionality)

## Screenshot

image
image

# Testing

Manual steps to reproduce this functionality:
Use navigation within CAT and the eMIB sample test.

# Checklist

Applicable for all code changes.

- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new compiler warnings
- [x] My changes generate no new accessibility errors and/or warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have researched WCAG2.1 accessibility standards and met them in this PR (can be navigated using a keyboard)
- [x] My changes look good on IE 11+ and Chrome

### CODE REVIEW

**Comment by MichaelCherry** [comment]
Still reviewing the code, but I noticed this in quit test: your state is initialized from a function with the Localize. So if you start to quit in EN; cancel; change language to FR; then click quit again, this happens: 
image

File: frontend/src/components/commons/QuitTest.jsx [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+    textAlign: "center",
+    marginTop: 32
+  }
+};
+
+const quitConditions = () => {
+  return [
+    { text: LOCALIZE.emibTest.testFooter.quitTestPopupBox.checkboxOne, checked: false },
+    { text: LOCALIZE.emibTest.testFooter.quitTestPopupBox.checkboxTwo, checked: false },
+    { text: LOCALIZE.emibTest.testFooter.quitTestPopupBox.checkboxThree, checked: false }
```
Comment (MichaelCherry) [inline frontend/src/components/commons/QuitTest.jsx]: Loading the LOCALIZE here causes the issue I noted in my other comment (i.e. that the language of the quit checkboxes is does not toggle language)

File: frontend/src/components/commons/QuitTest.jsx [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+    //reset all checkbox states on close
+    this.resetCheckboxStates();
+  };
+
+  openQuitPopup = () => {
+    this.setState({ showQuitPopup: true });
+  };
+
+  resetCheckboxStates = () => {
+    this.setState({ quitConditions: quitConditions() });
```
Comment (MichaelCherry) [inline frontend/src/components/commons/QuitTest.jsx]: hmm...  though this looks like a valid solution for the above problem.... interesting....

Comment (MichaelCherry) [reply frontend/src/components/commons/QuitTest.jsx]: ok, the issue only occurs IF YOU CLICK THE BACKGROUND to close the dialog. If you hit the button it works as expected....

**Comment by caleybrock** [comment]
@MichaelCherry the language toggle is an existing bug that I will log and fix in another PR - it isn't new from my code change.

File: frontend/src/components/commons/QuitTest.jsx (hunk 2 above)
Comment (MichaelCherry) [reply frontend/src/components/commons/QuitTest.jsx]: replicated on master... resolving

File: frontend/src/components/commons/QuitTest.jsx (hunk 1 above)
Comment (MichaelCherry) [reply frontend/src/components/commons/QuitTest.jsx]: replicated on master... resolving

**Pull request merged**